packages/block-library/src/query-loop/edit.js

```diff
@@ -41,6 +42,7 @@ export default function QueryLoopEdit( {
 			search,
 			exclude,
 			sticky,
+			useGlobalQuery,
```

Comment:
By convention `use*` is assigned to React hooks so it might be confusing.

Reply:
Thanks! Renamed `useGlobalQuery` to `isGlobalQuery` :+1: